tests/v2/unit/adapters/torch/lightning/test_dataset.py

```diff
@@ -73,14 +73,14 @@ def test_build_dataloader(self, mocker: MockerFixture) -> None:
             num_workers=2,
             pin_memory=False,
             shuffle=False,
-            drop_last=False,
+            drop_last=True,
```

Comment:
why this should be turned to True?

Reply:
I think this should only apply to classification, but it looks like I made a mistake. I'll revert. For reference, the issue on the classification side is https://github.com/openvinotoolkit/training_extensions/issues/2559, which has been fixed in develop but not yet reflected in feature/v2.

Thanks for the detailed review!

Reply:
Please attach the commit when you fix :)

Reply:
Yes, sure. https://github.com/openvinotoolkit/training_extensions/pull/2593/commits/f2a40cb1824809b649c15af7ae7bb00a3dfaa980 here is fixed commit :)